src/styles/DialogCatalog.scss

```diff
@@ -4,3 +4,7 @@
   position: relative;
   z-index: 0;
 }
+
+.mdc-dialog__content a:focus {
+  outline: 1px solid currentColor;
```

Comment:
I'm confused why this is needed... shouldn't focused links be outlined by default?

Reply:
They definitely _should_ be outlined by default, but for some reason Chrome doesn't render it on this page, whereas Firefox and Safari _do_. I don't want a silly Chrome bug to hold up our accessibility review.